cmd/ipfs/daemon.go

```diff
@@ -423,6 +446,35 @@ func serveHTTPGateway(req cmds.Request) (error, <-chan error) {
 	return nil, errc
 }
 
+type listing struct {
+	Uri  string
+	Keys []string
+}
+
+func loadKeySetFromURLs(urls []string) (key.KeySet, error) {
+	ks := key.NewKeySet()
+	for _, url := range urls {
+		resp, err := http.Get(url)
```

Comment:
where should we cache it to? just inside `$IPFS_PATH` ?
should we name them after the url used to fetch them? or just `blocklist.json` and `allowlist.json` ?